client/src/InfoBase/index_data.js

```diff
@@ -95,7 +95,7 @@ const index_lang_lookups = {
   },
   bonus_footer_list: {
```

Comment:
One catch here, this data's used to build the index html templates. Changes to those index html files have to be deployed to the IMTD server, we can't just push them out to the CDN. I can go through the process for that some time, just giving a heads up that this change won't take effect for a bit even after it's merged.

There's a currently disabled question on the FAQ page about how to contact us (disabled because we were waiting for a better email to make public). Could you re-enable that question and modify\* it to link to the contact route? Then there'd at least be one way to get there in the mean-time.

\* another little catch, the questions used on the FAQ route are loaded from a csv managed by the pipeline. To make any changes to that file permanent, the source file in the pipeline will also need to be updated. Just give me a heads up if you do edit that csv, I can mirror the changes to the pipeline before this merges :+1:.

Reply:
Ah, okay thanks for the heads up!
I've implemented the changes to the CSV; please do mirror the changes.